helm/ph-ee-engine/connector-gsma/values.yaml

```diff
@@ -0,0 +1,29 @@
+ingress:
+  apiversion: networking.k8s.io/v1
+
+service:
+  apiversion: "v1"
+
+secret:
+  apiversion: "v1"
+
+configmap:
+  apiversion: "v1" 
+
+enabled: true
+replicas: 1
+image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-gsma"
```

Comment:
all these images tobe parameterized to values, please create a ticket for the same

Reply:
It is parameterized in deployment. These are in values.yaml can't parameterize from here. I will keep it empty here.